### PR TITLE
UIDEXP-98: Create link to the files generated by exports partially completed

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**
## Purpose
Update the previous functionality by adding link for job execution where not all records were exported successfully.

## Issues
[UIDEXP-98](https://issues.folio.org/browse/UIDEXP-98)

## Screenshot
image

### CODE REVIEW

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 1]
```diff
@@ -86,8 +86,9 @@ const JobLogsContainer = props => {
   const getFileNameField = record => {
     const fileName = get(record.exportedFiles, '0.fileName');
     const isSuccessfulRecord = record.status === JOB_EXECUTION_STATUSES.SUCCESS;
+    const areAllJobsFailed = !record.progress.exported;
 
-    if (!isSuccessfulRecord) return fileName;
+    if (!isSuccessfulRecord || areAllJobsFailed) return fileName;
```
Comment (VictorSoroka1) [inline src/components/JobLogsContainer/JobLogsContainer.js:91]: We need remove the `!isSuccessfulRecord` check as in this case the Scenario 1 won't
```suggestion
    if (areAllJobsFailed) return fileName;
```

Comment (Vladyslav-Velytskyi) [reply src/components/JobLogsContainer/JobLogsContainer.js]: It looks logical, thanks!

File: src/components/JobLogsContainer/JobLogsContainer.js [hunk 2]
```diff
@@ -85,9 +85,9 @@ const JobLogsContainer = props => {
 
   const getFileNameField = record => {
     const fileName = get(record.exportedFiles, '0.fileName');
-    const isSuccessfulRecord = record.status === JOB_EXECUTION_STATUSES.SUCCESS;
+    const areAllJobsFailed = !record.progress.exported;
 
-    if (!isSuccessfulRecord) return fileName;
+    if (areAllJobsFailed) return fileName;
```
Comment (YevheniiMaltsev) [inline src/components/JobLogsContainer/JobLogsContainer.js:90]: @Vladyslav-Velytskyi @VictorSoroka1 I wonder if we really need to add an extra variable or just can use

`!record.progress.exported;`

As for me this condition is short and clear enough: If not exported - return fileName

What do you think?

Comment (Vladyslav-Velytskyi) [reply src/components/JobLogsContainer/JobLogsContainer.js]: Done, thanks, nice suggestion

**Pull request merged**